exercises/bracket-push/example.scala

```diff
@@ -0,0 +1,23 @@
+import scala.util.parsing.combinator.RegexParsers
+
+object Brackets extends RegexParsers {
```

Comment:
As I said, I like your solution very much (it even motivates me look into Parsers and sgf-parsing). :-)
I was (and still am) not quite sure about the exact role of these examples, though. People are not supposed to ever look at it, so is it more for making sure that the TestSuite is correct?
In this case you definitely can have some fun and not just repeat some standard solution you already know.  
In summary: I am totally fine with your solution. :-)